src/modules/account/ui/ActionField/ActionField.tsx

```diff
@@ -0,0 +1,54 @@
+import { VFC, ReactNode } from 'react';
+import ActionFieldLayout from './ActionFieldLayout';
+import ActionFieldButton from './ActionFieldButton';
+import {
+  ActionFieldProvider,
+  useActionField,
+} from '@modules/account/contexts/ActionFieldContext';
+
+type ActionFieldProps = {
+  label: string | ReactNode;
+  mainContent?: ReactNode;
+  alternativeContent?: ReactNode;
+  mainActionText?: string | ReactNode;
+  alternativeActionText?: string | ReactNode;
+  showActionButton?: boolean;
+};
+const ActionField: VFC<ActionFieldProps> = (props) => {
+  return (
+    <ActionFieldProvider>
+      <ActionFieldContent {...props} />
+    </ActionFieldProvider>
+  );
+};
+
+export default ActionField;
+
+const ActionFieldContent: VFC<ActionFieldProps> = ({
+  label,
+  mainContent = <></>,
```

Comment:
This is the same as `undefined` so I don't think a default value is necessary here.